src/js/calculator.js

```diff
@@ -0,0 +1,65 @@
+export class Calculator {
+    constructor(total) {
+        this.prevDigit = 0;
+        this.result = [];
+        this.resultSign = '=';
+        this.total = total;
+        this.operation = {
+            '+': (a, b) => a + b,
+            '-': (a, b) => a - b,
+            'X': (a, b) => a * b,
+            '/': (a, b) => Math.floor(a / b),
+        };
+        this.operatorType = Object.keys(this.operation);
+    }
```

Comment:
```suggestion
export class Calculator {
    prevDigit; result; resultSign; total; operation; operatorType;
    
    constructor(total) {
        this.prevDigit = 0;
        this.result = [];
        this.resultSign = '=';
        this.total = total;
        this.operation = {
            '+': (a, b) => a + b,
            '-': (a, b) => a - b,
            'X': (a, b) => a * b,
            '/': (a, b) => Math.floor(a / b),
        };
        this.operatorType = Object.keys(this.operation);
    }
```

이렇게 이 class에서 사용되는 멤버변수에 경우 class field에 미리 선언해주면 좋답니다!
그렇지 않으면 이 class 내에 어떤 멤버변수가 있는지 보는 사람 입장에서는 판단하기가 어려워요.